config/src/test/scala/com/lynbrookrobotics/potassium/config/Test.scala

```diff
@@ -9,7 +10,11 @@ object Test extends App{
   case class RobotConfig(drive: DriveConfig)
```

Comment:
While we're poking around here, do you think it would be possible to refactor this into a proper unit test?